sklearn/metrics/pairwise.py

```diff
@@ -1202,6 +1214,20 @@ def pairwise_distances(X, Y=None, metric="euclidean", n_jobs=1, **kwds):
     return _parallel_pairwise(X, Y, func, n_jobs, **kwds)
 
 
+# These distances recquire boolean arrays, when using scipy.spatial.distance
+PAIRWISE_BOOLEAN_FUNCTIONS = [
```

Comment:
Why not just import this, rather than re-defining it? e.g.
``from sklearn.metrics.pairwise import PAIRWISE_BOOLEAN_FUNCTIONS``